### PR TITLE
Add batch normalization in `FNOBlocks`

### PR DESCRIPTION
**Pull request by dhpitt**
* adds `BatchNorm` to `neuralop.layers.normalization_layers`
* adds `norm='batch_norm'` to `FNOBlocks.__init__()`

### CODE REVIEW

**Review by JeanKossaifi — APPROVED** [review]
Looks good, thank you!
